tests/formattedcode/data/yaml/package-and-licenses-expected.yaml

```diff
@@ -33,34 +33,7 @@ headers:
         platform_version: '#67-Ubuntu SMP Mon Mar 13 14:22:10 UTC 2023'
         python_version: "3.8.10 (default, Mar 13 2023, 10:26:41) \n[GCC 9.4.0]"
       spdx_license_list_version: '3.20'
-      additional_license_directory: /home/ayansinha/nexB/write_access/scancode-extra/tests/licensedcode/data/additional_licenses/additional_dir
-      additional_license_plugins:
-        - /home/ayansinha/nexB/write_access/scancode-extra/venv/lib/python3.8/site-packages/licenses_to_install1
-        - /home/ayansinha/nexB/write_access/scancode-extra/venv/lib/python3.8/site-packages/licenses_to_install2
       files_count: 4
-summary:
```

Comment:
The summary should probably appear first in the JSON before Packages/Deps/Licenses IMHO the previous order made a lot of sense, what do you think?

Reply:
@AyanSinhaMahapatra 

I think this is an effect of having the summary and/or file classification plugin run after the package post-scan step, now that file classification has been updated to handle resources with package data in them (for get_top_level_resources). 

@pombredanne 

Do you know of a way to order the top-level codebase attributes without changing the plugin order?